plothist/hep_plotters.py

```diff
@@ -17,6 +17,7 @@ def compare_data_mc(
     ylabel=None,
     mc_labels=None,
     mc_colors=None,
+    ratio="diff",
```

Comment:
The default is not a difference "diff", but a ratio. In the future we want to implement true difference or residual (data-MC).
We should not call the argument ratio, but `comparison` for example, that could take `ratio`, `pull`, or `diff`. In the future, we may even propose a combination of multiple comparisons with multiple bottom plots, like `["ratio", "pull"]`.

Now come the less fun part: in that direction, we should rename `ax_comparison` -> `ax_main` et `ax_ratio` -> `ax_comparison` for example, unless you have a better idea.